src/css/main.scss

```diff
@@ -1,7 +1,9 @@
-@import "variables";
-@import "~bootstrap/dist/css/bootstrap";
+@import "styles/variables";
+@import "styles/bootstrap";
+@import "~bootstrap/dist/css/bootstrap-grid";
```

Comment:
IMO we could get rid of `"~bootstrap/dist/css/bootstrap-grid"` and uncomment `@import "node_modules/bootstrap/scss/grid";` in `bootstrap.scss`